### DISCUSSION TITLE
SASL/SCRAM credentials and Multi-clusters

### DISCUSSION THREAD

**Discussion by baganokodo** [tchiotludo/akhq#1091] [Q&A] [answered]
Hi team,

"sasl.jaas.config" takes a clear text password in the application.yml file, which is security concern in the production environments. Any suggestion to hide the clear text credentials, especially in a multi-clustered environments?

Many thanks

**Comment by tchiotludo** [comment]
How do you deploy akhq? standalone, docker or k8s?

**Reply by baganokodo** [reply]
Hi @tchiotludo , it is deployed with docker-compose. thanks

**Comment by tchiotludo** [comment, marked as answer]
Maybe this one: https://www.howtogeek.com/devops/how-to-secure-sensitive-data-with-docker-compose-secrets/ ? 
I never used by myself

**Reply by baganokodo** [reply]
thanks very much, I will give it a try
